src/wazuh_db/wdb_parser.c

```diff
@@ -5378,7 +5391,7 @@ int wdb_parse_global_sync_agent_groups_get(wdb_t* wdb, char* input, char* output
             if (cJSON_IsTrue(j_get_hash)) {
                 get_hash = true;
             }
-            if (cJSON_IsNumber(j_agent_registration_delta)){
+            if (j_agent_registration_delta && cJSON_IsNumber(j_agent_registration_delta)){
```

Comment:
Although this check shouldn't be necessary, it was included to avoid having `scan-build` issues.

Reply:
We can remove the `cJSON_IsNumber()`check, at this point if the value isn't NULL, we know it has the right type of data